prebid-server/endpoints/openrtb2/auction-request-example.md

```diff
@@ -36,7 +36,7 @@ It serves 2 purposes:
                 "prebid": {                    // bid adapters don't see this object
                     "bidder": {
                         "bidderAlias": {       // bid adapters don't see this object
-                            "placement": 1001  // The contents of this object are seen by adapters at imp[].ext.bidder
+                            "placementId": 1001  // The contents of this object are seen by adapters at imp[].ext.bidder
```

Comment:
This example is weird anyways. Neither `placement` nor `placementId` are bidderAliases.

Can you elaborate how you found this and why this would fix the request example? @bretg any thoughts on this?

Reply:
This example is linked into the 'How to Build a PBS Adapter' pages. It's meant to provide a sense of what a PBS request looks like.

'bidderAlias' and 'placement' are completely fake.

@divshekhar - please explain what you're trying to accomplish? This isn't valid to POST to PBS no matter whether it's `placement` or `placementId`.

Reply:
@divshekhar ?